saleor/graphql/app/types.py

```diff
@@ -183,9 +205,19 @@ class App(ModelObjectType):
     )
     name = graphene.String(description="Name of the app.")
     type = AppTypeEnum(description="Type of the app.")
-    tokens = NonNullList(AppToken, description="Last 4 characters of the tokens.")
+    tokens = NonNullList(
+        AppToken,
+        description=(
+            "Last 4 characters of the tokens. Requires one of the following "
+            f"permissions: {AppPermission.MANAGE_APPS}"
```

Comment:
We should also include "OWNER" permission in this description, as owners can also fetch it.

Reply:
Could we use the same function across the entire core to format the "Requires (...)" message with consistent newlines etc.?